src/Lms/Canvas/CanvasLms.php

```diff
@@ -88,12 +88,9 @@ public function getLtiAuthUrl($globalParams)
 
     public function getKeysetUrl()
     {
-        if ($_ENV['JWK_BASE_URL']) {
-            $jwk_url_no_trailing_slash = rtrim($_ENV['JWK_BASE_URL'], '/');
-            return "{$jwk_url_no_trailing_slash}/api/lti/security/jwks";
-        }
         $session = $this->sessionService->getSession();
-        $baseUrl = $session->get('iss');
+        $baseUrl = $_ENV['JWK_BASE_URL'] ?: $session->get('iss');
```

Comment:
Same as mentioned before. I love the elvis operator, but in this case I think it's better to use `!empty()` since `JWK_BASE_URL` is an optional parameter.